biopet-utils/pom.xml

```diff
@@ -31,6 +31,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.typesafe.play</groupId>
```

Comment:
yes but try reading this ;)
https://www.implicitdef.com/2015/11/19/comparing-scala-http-client-libraries.html